docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,44 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+TestCafe actions can interact with elements if they satisfy the following conditions:
+
+* an element has to be within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element may be outside of a browser window- as long as TestCafe can reach it by scrolling.
```

Comment:
we can change 'has to be' to just 'is' because we have already said that these are conditions that should be satisfied.

* *an element is within...*

And I don't think we need a hyphen in the second sentence. Neither `-ing`.

*The element can be outside of a browser window as long as TestCafe can scroll to reach it.*

Reply:
I changed all the occurrences